packages/hub/node-tests/routes/profile-purchases-test.ts

```diff
@@ -143,7 +145,7 @@ describe('POST /api/profile-purchases', function () {
     let cardSpaceRecord = (await cardSpacesQueries.query({ merchantId }))[0];
     expect(cardSpaceRecord).to.exist;
 
-    let jobTicketRecord = await jobTicketsQueries.find({ id: jobTicketId! });
+    let jobTicketRecord = await prisma.jobTicket.findUnique({ where: { id: jobTicketId! } });
```

Comment:
I’m surprised to find that `prisma.jobTicket` and `prisma.JobTicket` both work here 🤔 maybe we should make only one of them work, but if we do, the build will fail so it shouldn’t hard to update everything as needed